### ISSUE TITLE
Safari No Local Video feed and black (or no) video feed on remote browser

### ISSUE THREAD

**Issue by edgolub** [twilio/twilio-video.js#946]
- [x] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [x] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [x] I verified that the [Twillio video react app](https://github.com/twilio/twilio-video-app-react) application works in my environment.
 - [x] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

I was using the [Twillio video react app](https://github.com/twilio/twilio-video-app-react) for comparing it against my own code, and found the same issue as in my app. I believe it is an issue with the twilio-video.js library because of it.

**Expected behavior:**
Well, it should connect two parties and show the video and audio feed on both sides.

**Actual behavior:**
When connecting via an iPhone (tested with latest version of Safari), there doesn't seem to be any local video feed, and, while I do get both a audio and video track on my remote device, the audio track is fine, while the video track reproduces as just a black video element.

No issues in log on my remote device. I don't own an iPhone myself so can't post a log. The same issue was seen in two different iPhones with the same Safari version though.

The issue does not happen on any Android device.
**Software versions:**

 - [x] Browser(s): Safari 13.1
 - [x] Operating System: iOS 13.4
 - [x] twilio-video.js: 2.3.0
 - [x] Third-party libraries (e.g., Angular, React, etc.): VueJs

**Comment by makarandp0** [comment]
Hello @edgolub, Thank you for writing about this issue.

Does this issue repro consistently for you? Can you provide more details - room-sid and client side logs would be most helpful. 

There are few known issues with iOS Safari. You can find more details on them here:
https://github.com/twilio/twilio-video.js/issues/941

Also there are few tweaks might be needed when working with mobile browsers. See some common issues we see and our recommendations @ https://www.twilio.com/docs/video/build-js-video-application-recommendations-and-best-practices

Thanks,
Makarand

**Comment by edgolub** [comment]
I believe this is the same issue as #927, so this is a duplicate.

What worked for me was setting the height explicitly. 720p worked, so I'd recommend what @bwilso65 tried. Worked for me at least.

I did have another issue after that, but was related to the stream somehow not being record able properly on chrome with the MediaRecorder API (and it worked). I think it might be the frameRate being set to 24? Not sure, but thankfully I just pushed the handful of users that use the recording to use Firefox and that was that for me.

I'll close this since it is a duplicate.
